src/styles/globalStyle.ts

```diff
@@ -0,0 +1,58 @@
+import { createGlobalStyle, css } from 'styled-components';
+
+import { type RiskLevels } from '@/constants/abacus';
+
+export const GlobalStyle = createGlobalStyle`
+  :root {
+    /* Computed: Layers */
+    --color-layer-0: ${({ theme }) => theme.layer0};
+    --color-layer-1: ${({ theme }) => theme.layer1};
+    --color-layer-2: ${({ theme }) => theme.layer2};
+    --color-layer-3: ${({ theme }) => theme.layer3};
+    --color-layer-4: ${({ theme }) => theme.layer4};
+    --color-layer-5: ${({ theme }) => theme.layer5};
+    --color-layer-6: ${({ theme }) => theme.layer6};
+    --color-layer-7: ${({ theme }) => theme.layer7};
+
+    /* Computed: Borders */
+    --color-border: ${({ theme }) => theme.borderDefault};
+    --color-border-white: ${({ theme }) => theme.borderButton}; //xcxc
+    --color-border-red: ${({ theme }) => theme.borderDestructive}; //xcxc
+
+    /* Computed: Text */
+    --color-text-0: ${({ theme }) => theme.textPrimary}; //xcxc
```

Comment:
Can we audit the text colors? `--color-text-0` was the tertiary before but I don't see other files changed reflecting the switch to from `--color-text-2` -> `--color-text-0`

Reply:
![image](https://github.com/dydxprotocol/v4-web/assets/13111994/deb43c26-f7b8-4f90-b18b-8f4077f8ff26)

In this image, the active tab has the tertiary color and the inactive tabs have the primary color :P

Reply:
oh interesting - I assumed text-0 mapped to primary, etc but bad assumption on my end. Will fix!!

Reply:
We can change it to that in a follow up since `0` being `primary` makes more sense